failing test on rpi

```
platform linux2 -- Python 2.7.9, pytest-3.0.6, py-1.4.32, pluggy-0.4.0 -- /home/pi/projects/pi-test/luma.lcd/.tox/py27/bin/python2.7
cachedir: .cache
rootdir: /home/pi/projects/pi-test/luma.lcd, inifile: 
plugins: cov-2.4.0
collected 12 items 

tests/test_backlight.py::test_unsupported_platform FAILED
tests/test_backlight.py::test_init PASSED
tests/test_backlight.py::test_enable_on PASSED
tests/test_backlight.py::test_enable_off PASSED
tests/test_pcd8544.py::test_init_84x48 PASSED
tests/test_pcd8544.py::test_hide PASSED
tests/test_pcd8544.py::test_show PASSED
tests/test_pcd8544.py::test_display PASSED
tests/test_st7735.py::test_init_160x128 PASSED
tests/test_st7735.py::test_hide PASSED
tests/test_st7735.py::test_show PASSED
tests/test_st7735.py::test_display PASSED

================================================================ FAILURES ================================================================
_______________________________________________________ test_unsupported_platform ________________________________________________________

    def test_unsupported_platform():
        with pytest.raises(luma.core.error.UnsupportedPlatform) as ex:
>           backlight(bcm_LIGHT=19)
E           Failed: DID NOT RAISE <class 'luma.core.error.UnsupportedPlatform'>

tests/test_backlight.py:30: Failed
================================================== 1 failed, 11 passed in 5.92 seconds ===================================================
ERROR: InvocationError: '/home/pi/projects/pi-test/luma.lcd/.tox/py27/bin/coverage run -m py.test -v'
```

--- a/luma/lcd/aux.py
+++ b/luma/lcd/aux.py
@@ -3,7 +3,7 @@
 # See LICENSE.rst for details.
 
 
-from luma.core import lib
+from luma.core import lib, error
 
 
 __all__ = ["backlight"]
@@ -22,8 +22,14 @@ class backlight(object):
     def __init__(self, gpio=None, bcm_LIGHT=18):
         self._bcm_LIGHT = bcm_LIGHT
         self._gpio = gpio or self.__rpi_gpio__()
-        self._gpio.setmode(self._gpio.BCM)
-        self._gpio.setup(self._bcm_LIGHT, self._gpio.OUT)
+
+        try:
+            self._gpio.setmode(self._gpio.BCM)
+            self._gpio.setup(self._bcm_LIGHT, self._gpio.OUT)
+        except RuntimeError as e:
+            if str(e) == 'Module not imported correctly!':
+                raise error.UnsupportedPlatform('GPIO access not available')
+
         self.enable(True)
 
     def enable(self, value):

--- a/setup.py
+++ b/setup.py
@@ -34,7 +34,7 @@ setup(
     namespace_packages=["luma"],
     packages=["luma.lcd"],
     zip_safe=False,
-    install_requires=["luma.core>=0.5.2"],
+    install_requires=["luma.core>=0.5.4"],
     setup_requires=pytest_runner,
     tests_require=test_deps,
     extras_require={

--- a/tests/test_backlight.py
+++ b/tests/test_backlight.py
@@ -8,8 +8,6 @@ try:
 except ImportError:
     from mock import Mock
 
-import pytest
-
 import luma.core.error
 from luma.lcd.aux import backlight
 
@@ -26,9 +24,10 @@ def setup_function(function):
 
 
 def test_unsupported_platform():
-    with pytest.raises(luma.core.error.UnsupportedPlatform) as ex:
+    try:
         backlight(bcm_LIGHT=19)
-    assert str(ex.value) == 'GPIO access not available'
+    except luma.core.error.UnsupportedPlatform as ex:
+        assert str(ex) == 'GPIO access not available'
 
 
 def test_init():

--- a/tests/test_backlight.py
+++ b/tests/test_backlight.py
@@ -25,7 +25,10 @@ def setup_function(function):
 
 def test_unsupported_platform():
     try:
-        backlight(bcm_LIGHT=19)
+        e = RuntimeError('Module not imported correctly!')
+        errorgpio = Mock(unsafe=True)
+        errorgpio.setmode.side_effect = e
+        backlight(bcm_LIGHT=19, gpio=errorgpio)
     except luma.core.error.UnsupportedPlatform as ex:
         assert str(ex) == 'GPIO access not available'
 

